src/index.html

```diff
@@ -5,8 +5,415 @@
+      rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a 
+        href="#" 
+        class="header__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo_img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+            
+          <li class="nav__item">
+            <a 
+              href="#laptops&computers" 
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops&nbsp;&amp;&nbsp;Computers
+            </a>
+          </li>
+          
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+          
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">
+              Tablets
+            </a>
+          </li>
+          
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">
+              Photo
+            </a>
+          </li>
+          
+          <li class="nav__item">
+            <a href="#video" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav> 
+    </header>
+
+    <main class ="main">
+      <div class="wrapper">
+        <div class="container" data-qa="card">
+          <img 
+            class="container__image"
+            src="./images/imac.jpeg" 
+            alt="Imac"
+          >
+    
+          <h2>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          
+          <div class="container__product-code">
+            Product code: 195434
+          </div>
+    
+          <div class="container__feedback">
+            <div class="stars">
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
+
+            <span class="container__reviews">
+              Reviews: 5
+            </span>
+          </div>
+    
+          <div class="container__price">
+            <span class="container__price-label">Price:</span>
+            <span class="container__price-value">$2,199</span>
+          </div>
+    
+          <a
+            class="container__button-buy"
+            data-qa="card-hover"
+            href="#"
+          >
+          BUY
```

Comment:
use lovercased value with adding uppercase style
```suggestion
           buy
```

Reply:
is not fixed. It should be one space before the word and word should be at lowercased. 
Apply all over the page